builders/get-started/networks/moonbeam-dev.md

```diff
@@ -111,6 +119,13 @@ To build the binary file, you can take the following steps:
         apt install clang protobuf-compiler libprotobuf-dev -y 
         ```
 
+        For MacOS users, these dependencies can be installed via homebrew:
+
+        ```bash
+        brew install llvm
+        brew install protobuf
+        ```
```

Comment:
I'm curious, did you run into an error that required you to install these on your mac? If so, do you recall what the exact error was?